homeassistant/components/firetv/media_player.py

```diff
@@ -161,11 +165,14 @@ def _adb_exception_catcher(self, *args, **kwargs):
 class FireTVDevice(MediaPlayerDevice):
     """Representation of an Amazon Fire TV device on the network."""
 
-    def __init__(self, ftv, name, get_sources):
+    def __init__(self, ftv, name, get_sources, apps):
         """Initialize the FireTV device."""
-        from firetv import KEYS
+        from firetv import APPS, KEYS
+        self.apps = APPS
         self.keys = KEYS
 
+        self.apps.update(dict(apps))
```

Comment:
We should be using vol to ensure apps is a dict already so we don't need to create a copy of it here

Reply:
So I can simplify it like this?

```python
self.apps.update(apps)
```

Reply:
I changed this to what I posted above, and it works. If that's not what you had in mind, let me know and I'll fix it.